src/model/ICommand.java

```diff
@@ -0,0 +1,8 @@
+package model;
+
+import java.io.IOException;
+
+public interface ICommand {
+    String doCommand(String arg) throws ClassNotFoundException, IOException;
+    String validateParams(String param);
```

Comment:
"doCommand should validate it's params first .."

Reply:
Great comment! fixed both in the interface and the implementation class.

Reply:
Hope I'll see that